fictionsuit/commands/basics.py

```diff
@@ -32,7 +33,7 @@ async def cmd_lorem(self, message, args):
         """)
 
 
-    async def cmd_summarize(self, message, args):
+    async def cmd_summarize(self, message, args, bot_id):
```

Comment:
This parameter doesn't appear to be being passed to the handler; uncommitted changes?

Reply:
Unrelatedly: since the bot's id doesn't change, it might be better to pass it into Basics in its constructor instead of passing it to each command invocation